templates/Element/FilterBox/filter_box.twig

```diff
@@ -1,7 +1,7 @@
 <div class="filter-box">
     {% if not hideFilter %}
     <div>
-        {{ element('FilterBox/filter_box_common') }}
+        {{ element('FilterBox/filter_box_common', { showFilterSearchByType }) }}
```

Comment:
```suggestion
        {{ element('FilterBox/filter_box_common', { 'showFilterSearchByType': true }) }}
```

Reply:
The variable `showFilterSearchByType` cannot take the value true inside filter_box because it inherits the value from the parent element, which in this case is `relations_add`.